src/snowflake/snowpark/modin/plugin/extensions/index.py

```diff
@@ -1426,21 +1426,20 @@ def value_counts(
         normalize: bool = False,
         sort: bool = True,
         ascending: bool = False,
-        bins: Any = None,
+        bins: int | None = None,
         dropna: bool = True,
-    ) -> native_pd.Series:
-        # how to change the above return type to modin pandas series?
+    ) -> Series:
```

Comment:
Is there a reason why we're not adding the docstrings under the docstrings directory to override Modin's docstrings similar to Series, DataFrame, ..etc?

Reply:
Since this is a new class which doesn't exist in modin, I am not sure if the approach will work, but @sfc-gh-vbudati can you create a ticket to give it a try, it would be great if we can centralize all documentations

Reply:
https://snowflakecomputing.atlassian.net/browse/SNOW-1542483 ticket here, I will try working on this soon